<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Medical Image Processor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>E:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EInput">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton">
        <property name="text">
         <string>Fill Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breaksCheckBox">
        <property name="text">
         <string>Breaks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomInButton">
        <property name="text">
         <string>Zoom In</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOutButton">
        <property name="text">
         <string>Zoom Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="undoButton">
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionRussian"/>
     <addaction name="actionGerman"/>
     <addaction name="actionFrench"/>
     <addaction name="actionChinese"/>
     <addaction name="actionJapanese"/>
     <addaction name="actionSpanish"/>
    </widget>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionLightTheme"/>
     <addaction name="actionDarkTheme"/>
    </widget>
    <widget class="QMenu" name="menuAdaptation">
        <property name="title">
            <string>OS Adaptation</string>
        </property>
        <addaction name="actionToggleNativeWidgets"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuTheme"/>
    <addaction name="menuAdaptation"/>
   </widget>
   <action name="actionEnglish">
    <property name="text">
     <string>English</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionRussian">
    <property name="text">
     <string>Русский</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionGerman">
    <property name="text">
     <string>Deutsch</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionFrench">
    <property name="text">
     <string>Français</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionChinese">
    <property name="text">
     <string>中文</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionJapanese">
    <property name="text">
     <string>日本語</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionSpanish">
    <property name="text">
     <string>Español</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionLightTheme">
    <property name="text">
     <string>Light</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionDarkTheme">
    <property name="text">
     <string>Dark</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </action>
   <action name="actionToggleNativeWidgets">
       <property name="text">
           <string>OS Adaptation</string>
       </property>
       <property name="checkable">
           <bool>true</bool>
       </property>
   </action>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
